DOL.WHD.Section14c.Web/index.html

```diff
@@ -7,7 +7,7 @@
 </head>
 <body>
   <div ng-view class="full-view"></div>
-  <script src='env.js'></script>
+  <!--<script src='env.js'></script>-->
```

Comment:
Go ahead and remove commented code.  Since we've got it versioned, if we need it again, it's in the history. :)